docs(js): Add dist to source maps troubleshooting page (#3943)
getsentry/sentry-docs: src/platforms/javascript/common/sourcemaps/troubleshooting_js.mdx

Alternately, instead of `sourceMappingURL`, you can set a `SourceMap` HTTP header on your minified file. If this header is present, Sentry will use it to discover the location of your source map.

<PlatformSection notSupported={["javascript.electron"]}>

> ## Verify artifacts distribution match value configured in your SDK

Reviewer: ```suggestion
## Verify artifact distribution value matches value configured in your SDK
```

> Whenever you are using distribution identifier (`dist` configuration option in the SDK), it has to be matched during the source maps upload as well.

Reviewer 2: ```suggestion
Whenever you are using a distribution identifier (`dist` configuration option in the SDK), the same value has to be used during the source maps upload as well.
```

Reviewer: Building on Abhi's suggestion (so this includes his and mine).

```suggestion
Whenever you are using a distribution identifier (the `dist` configuration option in the SDK), the same value has to be used during source map upload. Conversely, if your source maps are getting uploaded with a `dist` value, the same value must be set in your SDK.
```

> To do this, use `--dist` flag in the `sentry-cli` or `dist` option if you are using `@sentry/webpack-plugin`, and make sure that the same value is used in both places.

Reviewer: We just told them above that the values need to match, so I don't think we need to say it again.

```suggestion
To add a `dist` value to your uploaded source maps, use the `--dist` flag with `sentry-cli` or the `dist` option with `@sentry/webpack-plugin`. To set the `dist` value in the SDK, use the `dist` option in your `Sentry.init()`.
```

To verify that the distribution has been used and set correctly, open up the issue from the Sentry UI and check if the `dist` tag is present.

> For the artifacts, go to the `Source Maps` page in the project settings, and verify that required files have this value set. If not, it will say `none` instead.

Reviewer 2: ```suggestion
For artifacts, go to the `Source Maps` page in the project settings, and verify that the required files have this value set. If the value is not set, the files will say `none` instead.
```

> To verify that the distribution has been used and set correctly, open up the issue from the Sentry UI and check if the `dist` tag is present.
> For the artifacts, go to the `Source Maps` page in the project settings, and verify that required files have this value set. If not, it will say `none` instead.

Reviewer: The style guide now puts all sentences in a paragraph on the same code line, for better wrapping. Also, I don't think we need to mention "none" if we've told them where to look - it's pretty self-explanatory.

```suggestion
To verify that the distribution has been set correctly in the SDK, open an issue in the Sentry UI and check that the `dist` tag is present. For artifacts, go to the `Source Maps` page in project settings, choose the release shown on the event you just checked, and verify that the `dist` value (in the small oval next to the upload time) matches that on the event.
```
